packages/debugger/src/panels/sources/index.tsx

```diff
@@ -4,45 +4,53 @@
 |----------------------------------------------------------------------------*/
 
 import { IEditorServices } from '@jupyterlab/codeeditor';
-import { ITranslator, nullTranslator } from '@jupyterlab/translation';
+import { ITranslator } from '@jupyterlab/translation';
 import { ToolbarButton } from '@jupyterlab/ui-components';
-import { Panel } from '@lumino/widgets';
 import { viewBreakpointIcon } from '../../icons';
 import { IDebugger } from '../../tokens';
+import { SourcePathComponent } from './sourcepath';
 import { SourcesBody } from './body';
-import { SourcesHeader } from './header';
+import { ReactWidget } from '@jupyterlab/ui-components';
+import React from 'react';
+import { PanelWidget } from '../panelwidget';
 
 /**
  * A Panel that shows a preview of the source code while debugging.
  */
-export class Sources extends Panel {
+export class Sources extends PanelWidget {
   /**
    * Instantiate a new Sources preview Panel.
    *
    * @param options The Sources instantiation options.
    */
   constructor(options: Sources.IOptions) {
-    super();
+    super(options);
     const { model, service, editorServices } = options;
-    const translator = options.translator || nullTranslator;
-    const trans = translator.load('jupyterlab');
+    this.title.label = this.trans.__('Sources');
```

Comment:
Note: used do be singular `'Source'` (please ignore if the change is intentional)

Reply:
Nice catch, thanks! I thought the panel label takes the same name of class.